verif/accel_core/tests/accel_funcs.c

```diff
@@ -0,0 +1,12 @@
+// Define uint32_t as unsigned int
+#include "accel_core_defines.h"
+uint32_t *address = (uint32_t *)0x00FE0200;
+
+int xor_accel(int x, int y) {
+    int result;
+    WRITE_REG(address,x);
+    WRITE_REG((address+1), y);
+    READ_REG(result,address+2);
```

Comment:
Why READ_REG twice?

Reply:
We split it into two registers to write two operands. We can write them both in one cycle, the reason is that its much easier to debug and set data.  For example if I want to write 20x40 I can write 2040 at one reg but I found it much more difficult to debug and set data at first stages while debuging manually. After we will have the system working we can change it easiley. Anyway we will load weight automaticly